modules/apps/object/object-service/src/main/java/com/liferay/object/internal/field/business/type/PicklistObjectFieldBusinessType.java

```diff
@@ -139,11 +141,25 @@ public Map<String, Object> getProperties(
 				LocalizedValue localizedValue = new LocalizedValue(
 					objectFieldRenderingContext.getLocale());
 
-				localizedValue.addString(
-					objectFieldRenderingContext.getLocale(),
-					String.valueOf(
-						ObjectFieldSettingUtil.getDefaultValue(
-							null, objectField, null)));
+				Locale defaultLocale = objectFieldRenderingContext.getLocale();
+				String defaultValue = String.valueOf(
+					ObjectFieldSettingUtil.getDefaultValue(
+						null, objectField, null));
+
+				if (!StringUtil.equals(defaultValue, "null") &&
+					objectField.isLocalized()) {
```

Comment:
Can you replace `!StringUtil.equals(defaultValue, "null")` to use `Validator.isNotNull()` ?

attention to the condition order:
`if (objectField.isLocalized() && Validator.isNotNull(defaultValue)) {`

Reply:
Done!